src/main/java/core/basesyntax/figure/name/FigureNameSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.figure.name;
+
+import java.util.Random;
+
+public class FigureNameSupplier {
+    private Random random = new Random();
+
+    public String getRandomName() {
+        int index = random.nextInt(FigureName.values().length);
+        return FigureName.values()[index].name();

```

Comment:
This method is used only in switch - case. It's better to use enums in switch - case if possible
```suggestion
    public FigureName getRandomName() {
        int index = random.nextInt(FigureName.values().length);
        return FigureName.values()[index];
```